tests/importer_module.py

```diff
@@ -1,19 +1,7 @@
-import pytest
-import json
-from fastapi.testclient import TestClient
-
-# This is a pretty delicate thing, any change could cause it to break
-# Is there a better way to make this work?
-import sys
-
-from thor.main import app
-
-client = TestClient(app)
+# importer_module.py
 
+import json
 
-print("working")
-# Question: Would it be better to put the expected outputs into a separate file,
```

Comment:
Yes, I think creating a .txt file for the expected output would allow the code to be more readable and uniform.